presentation/intro/src/main/java/com/dhc/intro/splash/SplashScreen.kt

```diff
@@ -1,14 +1,33 @@
 package com.dhc.intro.splash
 
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 
 @Composable
-fun SplashScreen() {
-
+fun SplashScreen(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier,
+        contentAlignment = Alignment.Center,
+    ) {
+        // TODO : Lottie 로 변경
+        Box(
+            modifier = Modifier
+                .padding(bottom = 42.dp)
+                .size(204.dp)
+                .background(Color.Cyan),
+        )
+    }
 }
 
-@Preview
+@Preview(showBackground = true)
 @Composable
 private fun SplashScreenPreview() {
     SplashScreen()
```

Comment:
@JeonK1 
```suggestion
    DHCAndroidTheme {
        SplashScreen()
    }
```
nit) Preview 안에서는 Theme이 적용되지 않기 때문에 Theme으로 감싸줘도 좋을 것 같아. (우리 디자인이 머터리얼 기반은 아니지만 ...)

Reply:
반영완~ 이제보니 Theme, Color, Type 이 모두 app 모듈에 있어서 designsystem 모듈로 옮겼어! https://github.com/mash-up-kr/DHC-Android/pull/70/commits/7559cdb6a464b0c8ace12fabac937727913c55a3